azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -635,19 +637,32 @@ public static void deleteSymlinkedFile(final Path symlinkedFilePath)
     }
   }
 
+  /**
+   * Uploads the log file to the db for persistence.
+   * @param execId execution id of the flow.
+   */
+  private void uploadLogFile(final int execId) {
+    try {
+      this.executorLoader.uploadLogFile(execId, "", 0, FlowContainer.logFile);
```

Comment:
Why the name of the log file is emptyString?

Reply:
The name is only used for job log as per the code.
The logic was derived from FlowRunner where also the name is empty.